Formula/ncmpcpp.rb

```diff
@@ -31,6 +31,7 @@ class Ncmpcpp < Formula
   depends_on "pkg-config" => :build
   depends_on "libmpdclient"
   depends_on "readline"
+  depends_on "ncurses"
```

Comment:
https://github.com/arybczak/ncmpcpp/commit/5c62072713531dd4dd70aac73e675a884a4eaed7
https://github.com/arybczak/ncmpcpp/blob/5c62072713531dd4dd70aac73e675a884a4eaed7/NEWS#L16
This will be the case in 0.8+ 
"* Wide character version of ncurses is now required."
The head version would not have been buildable fir the last 5 months